packages/expo-modules-core/ios/Swift/DynamicTypes/DynamicEnumType.swift

```diff
@@ -4,7 +4,7 @@
  A dynamic type representing an enum that conforms to `Enumerable`.
  */
 internal struct DynamicEnumType: AnyDynamicType {
-  let innerType: Enumerable.Type
+  let innerType: any Enumerable.Type
```

Comment:
`any` is now required because the `CaseIterable` protocol has an associated type